swagger.json

```diff
@@ -5954,6 +5984,55 @@
                 "$ref": "#/definitions/DriverForCreate"
             }
         },
+        "createDvirParam": {
+            "name": "createDvirParam",
+            "description": "DVIR creation body",
+            "required": true,
+            "in": "body",
+            "schema": {
+                "type": "object",
+                "required": [
+                    "vehicleIsSafe"
+                ],
+                "properties": {
+                    "vehicleIsSafe" : {
```

Comment:
Would it make sense to split this DVIR model into a separate definition, and reference the DVIR object from `createDvirParam`?

Reply:
I don't see a benefit, since this is only used here, but if that's our convention I don't object!

Reply:
DVIR returned from the list endpoint, and the response from this POST, share a struct: `DvirBase`.

Reply:
I see, I think users of this API may be confused why the POST doesn't accept fields from the `DvirBase` model, such as `vehicleDefects`. Would it make sense to have this endpoint be a fully-fledged DVIR creation endpoint, such that it takes a superset of `DvirBase` (minus the DVIR `id` value).

Reply:
From a customer viewpoint, it would be nice to be able to generate full-fledged DVIRs.